samples/schema-registry-integration/pom.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
As we already have a `confluent-schema-registry-integration` should we prefix this w/ `scs-` (or some other prefix?).

Reply:
I am adding `scst-` prefix for all these artifacts.